deploy/prod/values.yaml

```diff
@@ -64,6 +64,41 @@ indexers:
     parameters:
       block_height_start: 1275799
       rpc_thread_blocks: 1
+  - network: ethereum
```

Comment:
We also need to add the secrets to Vault.

Reply:
Yes, RPCs to add:

- AVALANCHE
- BASE
- OPTIMISM
- ARBITRUM
- FANTOM

(seems like ETHEREUM and POLYGON already set)

Reply:
i'll add extra rpcs in vault, highlight also uses these blockchain

Reply:
All set.